apps/home/app/_shared/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,49 @@
+import { Email, People } from '@hamsurang/icon'
+import { Avatar, AvatarFallback, AvatarImage } from '@hamsurang/ui'
+
+type ProfileProps = {
+  name: string
+  username: string
+  email: string
+  description?: string
+  followers?: number
```

Comment:
```suggestion
  memberCount?: number
```
[기획]
저희 팔로워 대신 멤버로 표기하는 거 어떠신가요?!

Reply:
<img width="494" alt="image" src="https://github.com/user-attachments/assets/b9f8c5e9-5de7-413c-9247-8afcc1f52a12">

이 부분은 기획을 참고한건데, 기획 수정을 제안해주신걸까요?

Reply:
네넵 요거 아마 캡쳐 그대로 사용했던 걸루 기억해서요! 멤버로 표기 어떠신지 여쭤보았습니당

Reply:
전 좋아용 그럼 멤버로 바꾸겠습니다~!!

Reply:
저 근데 고민되는 게 있습니다. Profile 컴포넌트는 메인에서도 쓰이고, 멤버 페이지에서도 쓰이기 때문에
멤버 페이지에서 members 라는 prop을 갖는 것이 어색해요. 

1. 공감되신다면, 다른 제안 생각해보기
2. 아니면 followers 라는 영역 자체를 지우기 (어차피 People 에서 표기 되기 때문, 숫자가 중요한거라면 People 에서 숫자로도 표기)

<img width="502" alt="image" src="https://github.com/user-attachments/assets/6ae224d0-493e-41ab-8509-a2f3ae42a193">

예시

Reply:
완져니 죠아요 2번 좋습니다!!
- follwers 영역 자체 제거, People 옆에 숫자 표기

Reply:
[review: 리뷰 반영 (followers 삭제, people 숫자 추가)](https://github.com/hamsurang/origin/pull/25/commits/da44a0961dbec02e18be1de6cc84b8caed169b0c)
에서 작업 마무리했습니다

Reply:
Hey, Cool.